src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -4632,45 +4673,53 @@ namespace SimAirServingZones {
 						VpzHtgSumBySys( AirLoopNum ) += ZonePA;
 
 						// Fraction of required zone ventilation to minimum primary airflow expected at condition analyzed
-						FinalZoneSizing( CtrlZoneNum ).ZpzHtgByZone = 0.0;
+						TermUnitFinalZoneSizing( TermUnitSizingIndex ).ZpzHtgByZone = 0.0;
 						if ( VpzMinHtgByZone( CtrlZoneNum ) > 0 ) {
-							FinalZoneSizing( CtrlZoneNum ).ZpzHtgByZone = min( 1.0, FinalZoneSizing( CtrlZoneNum ).VozHtgByZone / VpzMinHtgByZone( CtrlZoneNum ) );
+							TermUnitFinalZoneSizing( TermUnitSizingIndex ).ZpzHtgByZone = min( 1.0, TermUnitFinalZoneSizing( TermUnitSizingIndex ).VozHtgByZone / VpzMinHtgByZone( CtrlZoneNum ) );
 						}
 
 						// calc zone primary air fraction
 						if ( ZoneSA > 0.0 ) Ep = ZonePA / ZoneSA;
 						if ( Ep > 1.0 ) Ep = 1.0;
-						FinalZoneSizing( CtrlZoneNum ).ZonePrimaryAirFractionHtg = Ep;
-						FinalZoneSizing( CtrlZoneNum ).ZoneOAFracHeating = ZoneOAFracHeating;
+						TermUnitFinalZoneSizing( TermUnitSizingIndex ).ZonePrimaryAirFractionHtg = Ep;
+						TermUnitFinalZoneSizing( TermUnitSizingIndex ).ZoneOAFracHeating = ZoneOAFracHeating;
 
 						// determined heated zone floor area in an airloop
 						for (int zoneInNode = 1; zoneInNode <= ZoneEquipConfig( CtrlZoneNum ).NumInletNodes; ++zoneInNode ) {
 							if ( ZoneEquipConfig( CtrlZoneNum ).InletNodeAirLoopNum( zoneInNode ) == AirLoopNum ) {
-								FinalSysSizing( AirLoopNum ).FloorAreaOnAirLoopHeated += Zone( ZoneEquipConfig( CtrlZoneNum ).ActualZoneNum ).FloorArea;
+								FinalSysSizing( AirLoopNum ).FloorAreaOnAirLoopHeated += DataHeatBalance::Zone( ZoneEquipConfig( CtrlZoneNum ).ActualZoneNum ).FloorArea;
 							}
 						}
+						TermUnitFinalZoneSizing( TermUnitSizingIndex ).SupplyAirAdjustFactor = max( ClgSupplyAirAdjustFactor, HtgSupplyAirAdjustFactor );
+
+						// Because Std62 calcs are currently structured by zone rather than airterminal, need to push information back into FinalZoneSizing
+						FinalZoneSizing( CtrlZoneNum ).VozHtgByZone = TermUnitFinalZoneSizing( TermUnitSizingIndex ).VozHtgByZone;
+						FinalZoneSizing( CtrlZoneNum ).ZpzHtgByZone = TermUnitFinalZoneSizing( TermUnitSizingIndex ).ZpzHtgByZone;
+						FinalZoneSizing( CtrlZoneNum ).ZonePrimaryAirFractionHtg = TermUnitFinalZoneSizing( TermUnitSizingIndex ).ZonePrimaryAirFractionHtg;
+						FinalZoneSizing( CtrlZoneNum ).ZoneOAFracHeating = TermUnitFinalZoneSizing( TermUnitSizingIndex ).ZoneOAFracHeating;
+						FinalZoneSizing( CtrlZoneNum ).SupplyAirAdjustFactor = TermUnitFinalZoneSizing( TermUnitSizingIndex ).SupplyAirAdjustFactor;
```

Comment:
But what if there is 2 air loops serving the same zone ? The last one will set this value.